### PR TITLE
Update pr_check component

### PR DESCRIPTION
**Pull request by mshriver**
should have removed this with the fix of the clowdapp template

Thanks for the contribution, take a moment to read our contributing and security guidelines:

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
IMHO, we should first merge #129

**Review by ezr-ondrej — APPROVED** [review]
I'd love to see the effect on the check (by merging 129 first), but I do not insist.

**Comment by mshriver** [comment]
the `ci.int` check for #129 is actually failing *because* this isn't merged. `bonfire deploy-iqe-cji` is failing because COMPONENT_NAME needs to match what's in app-interface for this repo.

File: pr_check.sh [hunk 1]
```diff
@@ -29,8 +29,7 @@ source $CICD_ROOT/build.sh
 source $CICD_ROOT/deploy_ephemeral_env.sh
 
 # Run smoke tests using a ClowdJobInvocation and iqe-tests
-COMPONENT_NAME="provisioning"  # name of app used in deploy-iqe-cji
```
Comment (ezr-ondrej) [inline pr_check.sh:32]: So should this be `provisioning-api`/`provisioning-backend`? what will `COMPONENT_NAME` be if we do not set it here?

Comment (mshriver) [reply pr_check.sh:32]: It's set at the top of the file to `provisioning-backend`, and should match whatever the component/clowdapp name is. This is what's used to do endpoint lookups in clowder envs for CJI pipelines.

**Review by ezr-ondrej — APPROVED** [review]
Thanks @mshriver ! :+1:

**Pull request merged**